easybuild/easyconfigs/d/daal/daal-2017.1.132.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = "IntelBase"
+name = 'daal'
```

Comment:
I suggest we actually use `DAAL` as software name? see https://software.intel.com/en-us/blogs/daal